tests/func/experiments/test_queue.py

```diff
@@ -35,65 +33,6 @@ def test_celery_logs(
     assert "failed to reproduce 'failed-copy-file'" in captured.out
 
 
-@pytest.mark.xfail(
-    strict=False,
-    reason="https://github.com/iterative/dvc/issues/9143",
-)
-def test_queue_remove_done(
```

Comment:
This case is covered by `tests/unit/repo/experiments/queue/test_remove.py::test_remove_done`. The unit test mocks out the celery calls that cause the flaky result in CI, and should still be sufficient for our needs (the general `exp remove` behavior is still tested elsewhere in our func tests).